You are using the **SAML auth backend**, but it is not properly
configured. Please check the following:


* You have set all the `SOCIAL_AUTH_SAML_*` settings in `{{ settings_path }}`
with the values from your SAML application.

* Navigate back to the login page and attempt the SAML auth flow again.
